anautilities.py

```diff
@@ -105,15 +97,8 @@ def isOutlierIQROneSided(arrayData, rejectHighTail=True):
 #See: https://github.com/joferkington/oost_paper_code/blob/master/utilities.py
 #Returns a boolean array with True if points are outliers and False otherwise.
 def isOutlierMAD(arrayData, thresh=3.5):
-    if len(arrayData.shape) == 1:
-        arrayData = arrayData[:,None]
-        pass
-
     median = np.median(arrayData, axis=0)
-
-    diff = np.sum((arrayData - median)**2, axis=-1)
```

Comment:
The problems @bdorney experienced when removing the reshaping code for 1D arrays was most likely caused by `np.sum` here. I simply removed this line because summing along the last axis was anyway not supported in `isOutlierIQR`.